### PR TITLE
⚙️Servo motor code for ESP32-S3

### PR DESCRIPTION
**Pull request by Meka04**
A code that rotates the servo motor from 30 to 150 degrees, and backwards.

### CODE REVIEW

File: Firmware/ESP32S3_Servo_Sweep
Comment (haloandrei) [inline Firmware/ESP32S3_Servo_Sweep:1]: Asta nu e un fisier de cod, ai adaugat un fisier binar. Incearca sa adaugi codul scris in C

**Review by haloandrei — CHANGES_REQUESTED** [review]

**Comment by haloandrei** [comment]
Adauga un emoji descriptiv ca prim caracter al titlului PR-ului. [Check this PR](https://github.com/ABC-practica-2024/BlackPearl/pull/6)

**Review by AporSzakacs — APPROVED** [review]
crazy

**Review by Andreescu-Arthur — APPROVED** [review]

**Review by 912-Cseke-Stefan — APPROVED** [review]

**Review by haloandrei — APPROVED** [review]

**Comment by haloandrei** [comment]
@Meka04 Green Light to merge when you are ready

**Pull request merged**